apps/docs/src/app/core/component-docs/popover/examples/popover-dialog/popover-dialog-example.component.html

```diff
@@ -1,7 +1,7 @@
 <button fd-button label="Open Dialog" (click)="openDialog(template)"></button>
 
 <ng-template #template let-dialog let-dialogConfig="dialogConfig">
-    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog">
+    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog" role="dialog">
```

Comment:
Do you really need `role="dialog"` here? `dialog.component.html` has the same role as well, looks unnecessary

Reply:
yes, we need this without 'role="dialog"' the contents are read multiple times, with this header content is read only once

Reply:
@Lokanathan-k nice to know the reason, thank you!
So we need to raise a separate issue, which would describe a double role for dialog component _(for `<fd-dialog>` and inside fd-dialog template)_ and it needs investigating (and resolve if possible)

Reply:
no need to raise issue, this issue resides in the Jaws Application for screen reading, on defining the role eliminates the duplicate reading.